Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure/token.go

```diff
@@ -1,12 +1,19 @@
 package azure
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"encoding/base64"
+	"fmt"
 	"net/http"
 	"net/url"
 	"strconv"
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
We need to re-vendor each time. That should do the trick. At least, that's how I understood it...